books/free-programming-books-pt_BR.md

```diff
@@ -353,11 +353,12 @@
 * [O (comovente) guia de Ruby do Why](http://why.carlosbrando.com)
 
 
-### Shell
+### Shell / Bash
```

Comment:
Please, give a reason for renaming

Reply:
I found the same on free-courses-fr.md and was trying to keep with the convention